### PR TITLE
fixing issue template for new guidance

### PR DESCRIPTION
**Pull request by MathewBiddle**
I just tried to use the issue template and it was broken. I think this should fix it.

### CODE REVIEW

**Comment by MathewBiddle** [comment]
Maybe we should do the same for the PR template too?

**Comment by ocefpaf** [comment]
> Maybe we should do the same for the PR template too?

Yep. I believe GH will deprecate the old format and we need to move to the new (yaml) one.

**Comment by MathewBiddle** [comment]
Think I'm done here. If it looks good feel free to merge.

File: .github/ISSUE_TEMPLATE/NEW_NOTEBOOK.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    id: example
+    attributes:
+      label: Module/Software/IOOS data service?
+      description: Can you provide a minimum example of the expected code and results in a notebook?
+      placeholder:
+  - type: input
+    id: description
+    attributes:
+      label: Provide a description.
+      description: Please provide a detailed description of the suggested example below
```
Comment (ocefpaf) [inline .github/ISSUE_TEMPLATE/NEW_NOTEBOOK.yml:36]: Just for consistency with the other descriptions.

```suggestion
      description: Please provide a detailed description of the suggested example below.
```

**Pull request merged**